### ISSUE TITLE
yamlscript.load returns str

### ISSUE THREAD

**Issue by 1Mark** [yaml/yamlscript#12]
I wanted to try https://yamlscript.org/posts/advent-2023/dec-21/ 
![image](https://github.com/yaml/yamlscript/assets/15360245/b8fd027e-cb81-4623-9a75-324d86598977)
But it kept printing `...`

Any ideas what's going on?

**Comment by 1Mark** [comment]
![image](https://github.com/yaml/yamlscript/assets/15360245/abfc433d-a407-49db-b04c-1165314d5479)
yamlscript.load returns str

**Comment by ingydotnet** [comment]
Hi @1Mark

Thanks for reading our blog and trying out YAMLScript with Python.

Looking over your screenshots I'm not sure what was causing your issue.

I made a reproduction script and it works for me here on Linux x64.
I'll try it on macos aarch64 in a minute.

Here's the script and results: https://gist.github.com/ingydotnet/34f8b3b279e81497bf4a97074aadbd75

I added this script to the repo here: https://github.com/yaml/yamlscript/blob/main/sample/issue-12.sh

Let me know if it works for you.
